.github/workflows/ci.yml

```diff
@@ -156,3 +156,30 @@ jobs:
 
       - name: Build Docker image
         run: ./run-tests.sh --check-docker-build
+
+  release-docker:
+    runs-on: ubuntu-20.04
+    if: >
+      vars.RELEASE_DOCKER == 'true' &&
+      github.event_name == 'push' &&
+      startsWith(github.ref, 'refs/tags/')
+    needs:
+      - lint-shellcheck
+      - lint-black
+      - lint-flake8
+      - lint-pydocstyle
+      - lint-check-manifest
+      - docs-sphinx
+      - python-tests
+      - lint-dockerfile
+    steps:
+      - name: Release docker image
```

Comment:
Cosmetics: let's capitalise as in "Release Docker image" for consistency with the other rules we have:

```console
$ rg Docker .github
• .github/workflows/ci.yml
159:      - name: Check Dockerfile compliance
168:      - name: Build Docker image
```